docker-for-windows/troubleshoot.md

```diff
@@ -265,19 +265,70 @@ Virtualization needs to be enabled in the BIOS. The steps to do so are Vendor
 specific, but typically the BIOS option is called `Virtualization Technology
 (VTx)` or similar.
 
+Once Hyper-V is enabled, it will show up as such on "Turn Windows features on or
+off".
+
+![Hyper-V on Windows features](images/hyper-v-enable-status.png )
+
 ### Virtualization must be enabled
 
 In addition to [Hyper-V](#hyper-v), virtualization must be enabled.
 
-If, at some point, if you manually uninstall Hyper-V or disable virtualization, Docker for Windows will not start.
+If, at some point, if you manually uninstall Hyper-V or disable virtualization,
+Docker for Windows will not start.
 
-Verify that virtualization is enabled on Task Manager.
+Verify that virtualization is enabled by checking the Performance tab on the
+Task Manager.
 
 ![Task Manager](images/win-virtualization-enabled.png)
 
 See also, the user reported issue [Unable to run Docker for Windows on Windows
 10 Enterprise](https://github.com/docker/for-win/issues/74)
 
+### Networking and Wifi problems upon Docker for Windows install
```

Comment:
Well, the only thing that would change is "Wifi", but I googled it and it's a proper noun. I corrected the capitalization on it. (Docker for Windows should also be initial caps, so that's okay as is.)